src/app/admin/settlement/[id]/page.tsx

```
@@ -327,7 +327,7 @@ export default function SettlementDetailPage() {
     return (
         <div className="flex flex-col mx-auto w-[var(--content-width)] py-8 gap-6">
             <div className="flex w-full">
-                <GhostButton onClick={() => router.push("/admin/settlement")} className="flex gap-1 items-center">
+                <GhostButton onClick={() => router.push("/admin/settlement")} className="flex gap-1 items-center" dataCy="back-to-list-button">
                     <ArrowLeftIcon className="w-4 h-4" />
                     목록으로
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: loadSettlement

This dependency is not specified in the hook dependency list.

Either include it or remove the dependency array

(lint/correctness/useExhaustiveDependencies)